agreement/persistence.go

```diff
@@ -16,7 +16,6 @@
 
 package agreement
 
-//go:generate dbgen -i agree.sql -p agreement -n agree -o agreeInstall.go
```

Comment:
This is a duplicate generation of the same file as below.

Reply:
nice catch